include/seqan3/alphabet/concept.hpp

```diff
@@ -182,7 +823,8 @@ template <CerealInputArchive archive_t, typename wrapped_alphabet_t>
 void CEREAL_LOAD_MINIMAL_FUNCTION_NAME(archive_t const &,
                                        wrapped_alphabet_t && l,
                                        alphabet_rank_t<detail::strip_cereal_wrapper_t<wrapped_alphabet_t>> const & r)
-    requires Semialphabet<detail::strip_cereal_wrapper_t<wrapped_alphabet_t>>
+    requires !std::is_const_v<std::remove_reference_t<wrapped_alphabet_t>> &&
```

Comment:
Doxygen

Reply:
that change was an unnecessary leftover anyway, I removed it.